multimodal/src/autogluon/multimodal/utils/metric.py

```diff
@@ -110,7 +111,7 @@ def infer_metrics(
             eval_metric_name = SPEARMANR
         else:
             eval_metric_name = RMSE
-    elif problem_type == OBJECT_DETECTION:
+    elif problem_type == OBJECT_DETECTION or problem_type == OPEN_VOCABULARY_OBJECT_DETECTION:
```

Comment:
`problem_type in [OBJECT_DETECTION, OPEN_VOCABULARY_OBJECT_DETECTION]` is better. Double check other places to see if such cases exist.

Reply:
Done.